tests/jballerina-semtype-port-test/src/test/resources/test-src/data/table-key.bal

```diff
@@ -13,8 +13,6 @@ type KS3 table<R> key(department);
 
 type KS4 table<R> key(city, department);
 
-// KS4<:KS5
-// KS5<:KS4
 type KS5 table<R> key(department, city);
```

Comment:
This is based on the last conversation with James. `table<C> key(f1,f2) != table<C> key(f2,f1)`